CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/).
 
+## [Unreleased] - 2023-09-30
```

Comment:
Sorry for being unclear about this thing

1. The release date is not deterministic, nix is in  an on-demand release mode
2. You probably should rebase your branch to master, so that you don't need to deal with that release date at all, and add your change log under [this log](https://github.com/nix-rust/nix/blob/57663c094848a80123794a7bd3ca88115fac055e/CHANGELOG.md?plain=1#L14), i.e., something like this:
     
    ```
    ### Changed

    - The following APIs now take an implementation of `AsFd` rather than a
    `RawFd`:

      - `unistd::tcgetpgrp`
      - `unistd::tcsetpgrp`
      - `unistd::fpathconf`
      - `unistd::ttyname`
      - `unistd::getpeereid`

      ([#2137](https://github.com/nix-rust/nix/pull/2137))
   
    - Changed `openat()` and `Dir::openat()`, now take optional `dirfd`s
      ([#2139](https://github.com/nix-rust/nix/pull/2139))
    ```

Reply:
I resolved the conflicts in the github web... When the PR merges/squashs in the main branch, all these intermediate commits will disappear.. It should not be a problem.